.mergify.yml

```
@@ -65,6 +77,7 @@ pull_request_rules:
     conditions:
       - base=master
       - label=automerge:rebase
+      - linear-history
```

Thinking about this more, I think can get away with allowing non linear history PRs to enter the merge queue if they're not up to date

```suggestion
      - or:
        - '#commits-behind>0'
        - linear-history
```